packages/ruby.rb

```diff
@@ -61,7 +62,14 @@ def self.install
 
   def self.postinstall
     puts 'Updating ruby gems. This may take a while...'
-    File.write("#{HOME}/.gemrc", "gem: --no-document\n", mode: 'a') if (File.exist?("#{HOME}/.gemrc") && !Kernel.system("grep -q \"gem: --no-document\" #{HOME}/.gemrc")) || !File.exist?("#{HOME}/.gemrc")
+    unless Kernel.system("grep -q \"no-document\" #{HOME}/.gemrc")
+      File.write("#{HOME}/.gemrc", "gem: --no-document\n",
+                   mode: 'a')
+    end
+    unless Kernel.system("grep -q \"gempath\" #{HOME}/.gemrc")
+      File.write("#{HOME}/.gemrc", "gempath: #{CREW_LIB_PREFIX}/ruby/gems/3.1.0\n",
```

Comment:
`#{CREW_DEST_HOME}`?

Reply:
This is meant to edit the gemrc in home, not in the destdir...

Reply:
The idea is to include `~/.gemrc` in the package files.

Reply:
I prefer the idea of leaving the HOME config files in postinstall, because that's ideally something which people might modify.

Also, since we're futzing about with the gemrc, it lets us avoid rebuilding the entire ruby package when we modify the gemrc.

Reply:
But maybe we could revisit for the release later this year of ruby 3.2?

Reply:
It probably doesn't matter anyway because if you `crew remove ruby`, crew stops working.  I was just thinking about the residue left over after removal but it's a moot point.